src/routes/transactions/mappers/common/native-staking.mapper.ts

```diff
@@ -205,26 +206,48 @@ export class NativeStakingMapper {
 
   private mapValidatorsExitStatus(
     networkStats: NetworkStats,
-    isConfirmed: boolean,
-    validatorsExitExecutionDate: Date | null,
+    transaction: MultisigTransaction | ModuleTransaction | null,
   ): StakingValidatorsExitStatus {
+    const isConfirmed =
+      transaction &&
+      'confirmations' in transaction &&
+      !!transaction.confirmations &&
+      transaction.confirmations.length >= transaction.confirmationsRequired;
+
     if (!isConfirmed) {
       return StakingValidatorsExitStatus.SignatureNeeded;
     }
 
-    if (!validatorsExitExecutionDate) {
+    if (!transaction.executionDate) {
       return StakingValidatorsExitStatus.AwaitingExecution;
     }
 
     // TODO: get validator status from the Kiln API
     const estimatedCompletionTime =
-      validatorsExitExecutionDate.getTime() +
+      transaction.executionDate.getTime() +
       networkStats.estimated_exit_time_seconds * 1000;
 
     return Date.now() <= estimatedCompletionTime
       ? StakingValidatorsExitStatus.RequestPending
       : StakingValidatorsExitStatus.ReadyToWithdraw;
   }
+
+  private getValueFromDataDecoded(data: DataDecoded, chain: Chain): number {
+    if (data.method === 'requestValidatorsExit') {
+      const publicKeys =
+        data.parameters?.filter(
+          (parameter) => parameter.name === '_publicKeys',
+        ) ?? [];
+      const decimals = chain.nativeCurrency.decimals;
+      return (
+        publicKeys.length *
+        Math.pow(10, decimals) *
+        NativeStakingMapper.ETH_ETHERS_PER_VALIDATOR
+      );
+    }
+
+    return 0;
+  }
```

Comment:
Nit: we should add a comment/test for this as it's very intricate.

Reply:
I've included some documentation here: https://github.com/safe-global/safe-client-gateway/pull/1884

Please let me know if you find it's not clear enough. Thanks!